src/ipc-handlers.ts

```diff
@@ -220,11 +220,9 @@ export async function updateSite(
 	return mergeSiteDetailsWithRunningDetails( userData.sites );
 }
 
-export async function connectWpcomSite(
-	event: IpcMainInvokeEvent,
-	sites: SyncSite[],
-	localSiteId: string
-) {
+type WpcomSitesToConnect = { sites: SyncSite[]; localSiteId: string }[];
+
+export async function connectWpcomSite( event: IpcMainInvokeEvent, list: WpcomSitesToConnect ) {
```

Comment:
```suggestion
export async function connectWpcomSites( event: IpcMainInvokeEvent, list: WpcomSitesToConnect ) {
```
Same thing here about potentially renaming.